### PR TITLE
Sameera donation levels

### PR DESCRIPTION
**Pull request by sameerav**
added in donation levels, fixed up a few project page styles, and changed the donation level model to have a textfield instead of a charfield!

should be a pretty short pull request! :)

### CODE REVIEW

File: revolv/static/project.scss [hunk 1]
```diff
@@ -265,6 +268,11 @@ body {
             font-size: 1.8rem;
         }
     }
+
+    .columns {
```
Comment (noahsark769) [inline revolv/static/project.scss]: Can you make this a different class name? I think that in general applying styles to foundation-specific classes is a little bit like using global variables: easier to understand how things work if you use local support instead :)

File: revolv/static/project.scss [hunk 2]
```diff
@@ -306,11 +314,57 @@ body {
...
+            color: #888888;
+
+            .rewards-text-inside {
+                font-size: 0.8rem;
+            }
+        }
+
+        .donation-level-box {
+            width: 95%;
+            //height: 400px;
```
Comment (noahsark769) [inline revolv/static/project.scss]: Can just delete :)

File: revolv/templates/project/project.html [hunk 3]
```diff
@@ -10,6 +10,8 @@
     <script type="text/javascript">
         window.PARTIAL_COMPLETENESS = {{ project.partial_completeness_as_js }}
     </script>
+
+    <link href='http://fonts.googleapis.com/css?family=Lato:100,300,400,700,800' rel='stylesheet' type='text/css'>
```
Comment (noahsark769) [inline revolv/templates/project/project.html]: Do you need all these font weights? The download for this css file will be larger than needed if we don't use them :(

File: revolv/templates/project/project.html [hunk 4]
```diff
@@ -77,12 +80,32 @@
...
+    <div class="large-4 columns rewards-container">
+        <div class="rewards-text"><p class="rewards-text-inside">REWARDS</p></div>
+        <div class="gray-bar"></div>
+        {% for donation_level in project.donation_levels %}
+            <div class="donation-level-box">
+                <div class="donation-level-header">${{ donation_level.amount }}</div>
+                <div class="donation-level-text">{{ donation_level.description }}</div>
+            </div>
+        {% endfor %}
+            <!-- UNCOMMENT THESE BOXES TO SEE SAMPLE DONATION LEVELS  -->
```
Comment (noahsark769) [inline revolv/templates/project/project.html]: This is a lot of comments, how do you feel about that? Seems a little unprofessional now that I think about it?

**Comment by noahsark769** [comment]
Should there be a migration for changing the description to a TextField?

**Comment by noahsark769** [comment]
Also, this is weird, but I can't see the donation levels on the project page after adding them...does it work for you?

**Comment by sameerav** [comment]
@noahsark769 added in the migration file and you should be able to see the donation levels on the project page now! it might have been the lack of the migration file that stopped you from seeing them

File: revolv/templates/project/project.html (hunk 4 above)
Comment (sameerav) [inline revolv/templates/project/project.html]: Yeah I agree :(
I deleted them

**Comment by noahsark769** [comment]
Things to do:

1. Get rid of `org_about_lines`
2. Smaller donate button text, more padding, more margin-bottom for donate button on mobile
3. Iframe has weird proportions on mobile
4. Rewards should highlight, open up donation modal ala indiegogo

File: revolv/templates/project/project.html [hunk 5]
```diff
@@ -10,6 +10,11 @@
     <script type="text/javascript">
         window.PARTIAL_COMPLETENESS = {{ project.partial_completeness_as_js }}
     </script>
+
+    <link href='http://fonts.googleapis.com/css?family=Lato:300,500,800' rel='stylesheet' type='text/css'>
```
Comment (noahsark769) [inline revolv/templates/project/project.html]: Move this up with the other stylesheets :)

File: revolv/templates/project/project.html [hunk 6]
```diff
@@ -60,13 +65,14 @@
     {% endif %}
 </div>
 
-
+{% if project_donation_levels|length > 0 %}
```
Comment (noahsark769) [inline revolv/templates/project/project.html]: Things inside this tag should be indented :)

**Comment by noahsark769** [comment]
3 things still:

1. The iframe proportions seem to be wrong on tablet (both orientations): 
screen shot 2015-04-25 at 7 11 53 pm
2. Same thing on mobile landscape: 
screen shot 2015-04-25 at 7 14 16 pm
3. The donation level dollar amounts don't change color when you hover over the box, only when you actually hover over the title. I think they should change whenever you hover over the entire box?

Other than that, should be good to go

**Comment by sameerav** [comment]
@noahsark769 fixed those things!

**Comment by noahsark769** [comment]
Yay! Looks great, thanks @sameerav!

**Pull request merged**